src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -20,6 +24,8 @@
 	private final String packageName;
 	private final Set<Path> sources = new LinkedHashSet<>();
 	private boolean overwrite = false;
+	private final ByteArrayOutputStream streamError = new ByteArrayOutputStream(256);
+	private final ByteArrayOutputStream streamOutput = new ByteArrayOutputStream(256);
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' has incorrect indentation level 8, expected level should be 2\.